docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # Uncomment the below if you use native CircuitPython modules such as
 # digitalio, micropython and busio. List the modules you use. Without it, the
 # autodoc module docs will fail to generate with a warning.
-# autodoc_mock_imports = ["digitalio", "busio"]
+autodoc_mock_imports = ["digitalio", "busio", "micropython"]
```

Comment:
Apparently this one got lost on my initial review too. `busio` not needed.

Reply:
thanks, fixed!